app/views/unattended/report_templates/ansible_-_ansible_inventory.erb

```diff
@@ -111,14 +111,14 @@ model: ReportTemplate
 require:
 - plugin: foreman_ansible
   version: '3.1.0'
```

Comment:
Also, I am questioning this require, the template renders just fine on a plain Foreman 2.1.4 for me.

Reply:
The template can just work as any other, but this template was added as part of Ansible Tower integration improvement. Foreman Ansible plugin contains an endpoint to be used from Tower and it uses this template to render inventory. Although nothing from Ansible plugin is used within the template, the plugin itself requires this template...

@ares can provide more details I guess.

Reply:
Right, foreman ansible provides a specific (stable) endpoint to call this, but there is no reason not to be able to also call it using the generic report endpoint (one would just need to know the right template id). And I think we should allow that?

But that's not the main point of this PR ;)

Reply:
I was under the impression we also needed some safemode changes from newer foreman_ansible plugin, however I may totally be wrong. I think @evgeni is right in his logic and this requirement can be dropped

Reply:
I've only tested this with Foreman 2.1 and (with and without) foreman_ansible 5.1.

I don't see anything safe mode related in https://github.com/theforeman/foreman_ansible/compare/v3.0.0...v4.0.0 tho

Reply:
Should we drop it in this PR, or later?

Reply:
I'd prefer followup PR, as this seems to be quite ready to go in, if we lift the ansible requirement, I'd argue it would be nice to have simple means to render it, what would call for improvements on the report render API.

Reply:
Ack, works for me.